packages/rest/src/__tests__/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -393,6 +408,137 @@ describe('RestServer.getApiSpec()', () => {
     expect(spec.info).to.eql(EXPECTED_SPEC_INFO);
   });
 
+  context('options', () => {
+    it('disables consolidator if consolidate is set to false', async () => {
+      const options: {rest: RestServerConfig} = {
+        rest: {openApiSpec: {consolidate: false}},
+      };
+      options.rest = givenHttpServerConfig(options.rest);
+      app = new Application(options);
+      app.component(RestComponent);
+      server = await app.getServer(RestServer);
+      await server.start();
+
+      server.api(
+        anOpenApiSpec()
+          .withOperation(
+            'get',
+            '/',
+            anOperationSpec().withResponse(200, {
+              description: 'Example',
+              content: {
+                'application/json': {
+                  schema: {
+                    title: 'loopback.example',
+                    properties: {
+                      test: {
+                        type: 'string',
+                      },
+                    },
+                  },
+                },
+              },
+            }),
+          )
+          .build(),
+      );
+
+      const EXPECTED_SPEC = anOpenApiSpec()
+        .withOperation(
+          'get',
+          '/',
+          anOperationSpec().withResponse(200, {
+            description: 'Example',
+            content: {
+              'application/json': {
+                schema: {
+                  title: 'loopback.example',
+                  properties: {
+                    test: {
+                      type: 'string',
+                    },
+                  },
+                },
+              },
+            },
+          }),
+        )
+        .build();
+
+      const spec = await server.getApiSpec();
+      expect(spec).to.eql(EXPECTED_SPEC);
+
+      await server.stop();
```

Comment:
We usually put `server.stop()` in the `afterEach` or `after` hook, thanks for adding it in the test, could you refactor it into the `afterEach` hook? will benefit other tests in this file.